show-versions.mjs

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/env npx zx
+
+import { join } from 'path'
+import semver from 'semver'
+import Table from 'cli-table'
+$.verbose = false
+
+const packages = JSON.parse(await $`lerna list --all --json --loglevel=silent --toposort`)
```

Comment:
Nice way of utilizing `--json` flag 👍

Reply:
Note the silly thing needed `--loglevel=silent` to stop it from spitting out non-json logs.